<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainabir</class>
 <widget class="QMainWindow" name="mainabir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>671</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>651</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>471</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>561</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>231</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>type de service:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>150</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>180</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>date de debut</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>date de fin</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Cancel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>300</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font-family: 'Sans-Serif';
 font-size: 26px;    
	background-color: rgb(209, 255, 117);
border-radius:15px;


}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>matricule</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>cin_client</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>391</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>311</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{
	background-color: rgb(255, 255, 255);
border-radius:25px;

}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color:rgb(113, 113, 113);</string>
       </property>
       <property name="text">
        <string>ADD clients</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>71</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>POSTE:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>SEXE:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>DATE_NAISSANCE: </string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>230</y>
         <width>110</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TELEPHONE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN_CLIENT:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>MATRICULE</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>180</y>
         <width>80</width>
         <height>26</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>female</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>male</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color:transparent;
border:4px solid rgb(0, 0, 0);
border-radius:10px;



}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>140</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color:transparent;
border:4px solid rgb(0, 0, 0);
border-radius:10px;



}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>270</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color:transparent;
border:4px solid rgb(0, 0, 0);
border-radius:10px;



}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color:transparent;
border:4px solid rgb(0, 0, 0);
border-radius:10px;



}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>320</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color:transparent;
border:4px solid rgb(0, 0, 0);
border-radius:10px;



}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>380</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>submit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_87">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>430</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../AUTOGENIUS/AUTOGENIUS/images/locals.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>100</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color:transparent;
border:4px solid rgb(0, 0, 0);
border-radius:10px;



}</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>PRENOM</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>311</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{
	background-color: rgb(255, 255, 255);
border-radius:25px;

}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color:rgb(113, 113, 113);</string>
       </property>
       <property name="text">
        <string>Delete clients</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN:_CLIENT</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color:transparent;
border:4px solid rgb(0, 0, 0);
border-radius:10px;



}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_12">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>380</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>delete</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="afficher_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font-family: 'Sans-Serif';
 font-size: 26px;    
	background-color: rgb(209, 255, 117);
border-radius:15px;


}</string>
    </property>
    <property name="text">
     <string>scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font-family: 'Sans-Serif';
 font-size: 26px;    
	background-color: rgb(209, 255, 117);
border-radius:25px;


}</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refrech">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>540</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font-family: 'Sans-Serif';
 font-size: 26px;    
	background-color: rgb(209, 255, 117);
border-radius:15px;


}</string>
    </property>
    <property name="text">
     <string>refrech</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trier">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}

QPushButton:hover {
    background-color: rgba(0, 0, 0, 0.1); /* Ajouter une légère couleur de survol */
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 0.2); /* Couleur lors du clic */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/105-removebg-preview.png</normaloff>:/105-removebg-preview.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>59</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font-family: 'Sans-Serif';
 font-size: 26px;    
	background-color: rgb(209, 255, 117);
border-radius:25px;


}</string>
    </property>
    <property name="text">
     <string>Modify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stat">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font-family: 'Sans-Serif';
 font-size: 26px;    
	background-color: rgb(209, 255, 117);
border-radius:25px;


}</string>
    </property>
    <property name="text">
     <string>STAT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RECHERCHE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="appel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font-family: 'Sans-Serif';
 font-size: 26px;    
	background-color: rgb(209, 255, 117);
border-radius:15px;


}</string>
    </property>
    <property name="text">
     <string>appeler</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>540</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font-family: 'Sans-Serif';
 font-size: 26px;    
	background-color: rgb(209, 255, 117);
border-radius:15px;


}</string>
    </property>
    <property name="text">
     <string>export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rechercher">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}

QPushButton:hover {
    background-color: rgba(0, 0, 0, 0.1); /* Ajouter une légère couleur de survol */
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 0.2); /* Couleur lors du clic */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/110-removebg-preview.png</normaloff>:/110-removebg-preview.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>67</width>
      <height>54</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font-family: 'Sans-Serif';
 font-size: 26px;    
	background-color: rgb(209, 255, 117);
border-radius:25px;


}</string>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>150</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>CIN_CLIENT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NOM</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>150</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TRIE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="afficher">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>540</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font-family: 'Sans-Serif';
 font-size: 26px;    
	background-color: rgb(209, 255, 117);
border-radius:15px;


}</string>
    </property>
    <property name="text">
     <string>afficher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calender">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font-family: 'Sans-Serif';
 font-size: 26px;    
	background-color: rgb(209, 255, 117);
border-radius:25px;


}</string>
    </property>
    <property name="text">
     <string>CALENDER</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1134</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
